### PR TITLE
Merge backlog Halil (detailsCard)

### PR DESCRIPTION
**Pull request by Halilurrahman**
footer update, detailcard size is responsive, content nog niet. Alle data is verbonden met backend

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "./App.css";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Row from "./Row";
 import requests from "./requests";
-import Hero from "./components/Hero";
+// import Hero from "./components/Hero";
```
Comment (solid5nake) [inline src/App.js]: With next merge remove class component Hero

**Review by solid5nake — APPROVED** [review]
Approved, Fetch is back in the Banner component

File: src/components/MoreInfoButton/InfoButton.js [hunk 2]
```diff
@@ -6,70 +6,101 @@ function InfoButton(props) {
   const [showMovie, setshowMovie] = useState(false);
   const toggleMovie = () => {
     setshowMovie(true);
+    var stringOfGenres = "asdasdf";
```
Comment (solid5nake) [inline src/components/MoreInfoButton/InfoButton.js]: Should be removed

File: src/components/MoreInfoButton/InfoButton.js [hunk 3]
```diff
@@ -4,35 +4,29 @@ import "./InfoButton.css";
 
 function InfoButton(props) {
   const [showMovie, setshowMovie] = useState(false);
+  const [genres, setGenres] = useState(null);
   const toggleMovie = () => {
     setshowMovie(true);
-    var stringOfGenres = "asdasdf";
```
Comment (solid5nake) [inline src/components/MoreInfoButton/InfoButton.js]: Good no bananas

**Review by Daniel-hospitality — COMMENTED** [review]
Looks good

File: src/components/MoreInfoButton/InfoButton.js [hunk 4]
```diff
@@ -89,14 +83,15 @@ function InfoButton(props) {
             <div className="speelduur_overview">
               <p>Runtime: {props.movie?.runtime} min </p>{" "}
               <p>{props.movie?.overview}</p>
-            </div>
-            <div className="director_genre">
-              {" "}
               <p>Director: {props.movie?.director}</p>
               <p>
-                Genres:  {props.movie?.genres[0].name}
+                Genres: {genres}
```
Comment (solid5nake) [inline src/components/MoreInfoButton/InfoButton.js:86]: useState Genres used

File: src/components/MoreInfoButton/InfoButton.js [hunk 5]
```diff
@@ -11,16 +11,14 @@ function InfoButton(props) {
...
-   let genres= "";
-   let values= "";
-  for (let i = 0; i < props.movie?.genres.length; i++) {
-       genres = genres.concat(props.movie?.genres[i].name + ", ");
-        }              
-        values =  genres.slice(0, -2);
-    
-  return values;
-  }
+  const filterGenres = () => {
```
Comment (solid5nake) [inline src/components/MoreInfoButton/InfoButton.js:14]: returnGenres() function refactored

**Review by solid5nake — APPROVED** [review]
Reviewed all and approve merge

**Comment by Daniel-hospitality** [comment]
Net04 Footer, Infobutton Backlog Approved

**Pull request merged**